VitvitskyAkim/HW7/task7_1.py

```diff
@@ -0,0 +1,4 @@
+def the_largest_number(num1,num2):
```

Comment:
good job
look at this implementation:
def return_max_number(first_number, second_number):
    '''
    This function returns the max number from two entered number
    '''
 
    first_number = float(first_number) 
    second_number = float(second_number)
 
    if first_number == second_number:
        return f"{first_number} is equal to {second_number}"
    if first_number > second_number:
        return f"Max number is {first_number}, min number is {second_number}"
    else:
        return f"Max number is {second_number}, min number is {first_number}"